### DISCUSSION TITLE
Node.js debugger has a default value of --inspect-brk=PORT without specifying this in the script.

### DISCUSSION THREAD

**Discussion by AdeleCloud** [babel/babel#14355] [Q&A]
Hi all, 

I am pretty new to JS and node and i couldn't find any info regarding the next. I will appreciate your help.

When I run my mocha test in debug mode Node.js debugger has a default value of --inspect-brk=PORT without specifying this in the script. My test script look like that
--timeout=15000 --appName=ejbrowser --role=adminUserOne --env=tst --reporter=spec tests/api/ejbrowser/devices/all.spec.js --failureFatal=1

But when I run it with debug mode the script that is being executed is:
C:\Program Files\nodejs\node.exe" C:\repos\yyc-api-ui-automation\node_modules\mocha\bin\mocha **--inspect-brk=57629** --appName=ejbrowser --role=adminUserOne --env=tst --timeout=60000 --slow=2000 --reporter=mochawesome tests/ui/-common/-health-checks-lvl1/all.spec.js --browserClient=chrome --failureFatal=1 --timeout 0 --ui bdd --reporter "C:\Program Files\JetBrains\IntelliJ IDEA 2021.3\plugins\NodeJS\js\mocha-intellij\lib\mochaIntellijReporter.js"

Where does this value  --inspect-brk=57629 come from?

The framework that I use doesn't accept this parameter in commandLineArgs and I had to modify it to accept { partial: true} which is not perfect. So I am trying to understand what is populating this value.

package,json uses this dependencies:
"@babel/cli": "^7.13.0",
"@babel/core": "^7.13.8",
"@babel/node": "^7.13.0",
"@babel/plugin-transform-runtime": "7.12.13",
"@babel/preset-env": "^7.13.9",
"@babel/register": "^7.13.8",
"base-64": "^1.0.0",
"browserstack-local": "^1.4.8",
"chai": "^4.3.3",
"command-line-args": "^4.0.6",
"deferred": "^0.7.11",
"dotenv": "^8.2.0",
"faker": "^5.4.0",
"fs-extra": "^9.1.0",
"iedriver": "^3.150.1",
"jira-client": "^6.21.1",
"jshint": "^2.12.0",
"jsonfile": "^6.1.0",
"mocha": "^9.2.1",
"mochawesome": "^7.1.2",
"moment": "^2.29.1",
"npm": "^8.5.4",
"rhea": "^1.0.24",
"selenium-webdriver": "^4.1.1",
"supertest": "^6.1.3"

This is the .babelrc file in the project
{
    "presets": ["@babel/preset-env"],
    "plugins": [
        ["@babel/transform-runtime", {
            "regenerator": true
        }]
    ]

<img width="726" alt="2022-03-14 09_35_48-yyc-api-ui-automation – all spec js" src="https://user-images.githubusercontent.com/99425818/158222807-4b894c87-65d6-400d-bb1f-8fd104977791.png">
}

**Comment by nicolo-ribaudo** [comment]
That flag is not related to Babel, but it's probably set by your debugging tool (probably your editor). I suggest asking in their support channels, or on Stack Overflow!

**Reply by AdeleCloud** [reply]
Thank you i will follow your advice and ask in Stack Overflow, I did posted a question in IntelliJ support page as well, but didn't receive reply yet.
